react-native/react/menubar/index.render.desktop.js

```diff
@@ -25,30 +28,19 @@ const Header = props => {
   const openKBFS: () => void = props.openKBFS
   const showUser: () => void = props.showUser
 
-  return (
-    <div style={styles.header}>
-      <i className={`fa fa-folder`} style={{...styles.icons, fontSize: 17, marginRight: 10}} onClick={openKBFS}/>
-      <i className={`fa fa-globe`} style={{...styles.icons, fontSize: 16}} onClick={showUser}/>
-    </div>
-  )
-}
-
-const OpeningMessage = props => {
-  const message: string = props.message
-
-  if (!message) {
-    return null
-  }
-
-  const buttonInfo: ?{
-    text: string,
-    onClick: () => void
-  } = props.buttonInfo
+  // $FlowIssue ignore
+  const version = __VERSION__ // eslint-disable-line no-undef
 
   return (
-    <div style={styles.openingMessage}>
-      <Text style={{textAlign: 'center', marginTop: 18, marginBottom: 11}} type='Body' reversed>{message}</Text>
-      {buttonInfo && <Button style={{marginBottom: 22}} onClick={buttonInfo.onClick} label={buttonInfo.text}/>}
+    <div style={styles.header}>
+      <Icon hint='Open KBFS folder' type='fa-folder' style={{marginRight: 10}} onClick={openKBFS}/>
+      <Icon hint='Open keybase.io web' type='fa-globe' onClick={showUser}/>
+      <div style={{flex: 1}}/>
+      <Icon hint={`Report a bug for version: ${version}`} type='fa-bug' onClick={ () => {
+        clipboard.writeText(`Keybase GUI Version: ${version}`)
+        shell.openExternal('https://github.com/keybase/client/issues')
```

Comment:
Could consider opening https://github.com/keybase/client/issues/new instead -- since client is a monorepo, having users try to browse existing issues before filing one isn't as helpful.

Oh!  We can even prepopulate fields on that form.  So instead of writing the version to the clipboard, we can just add it to the page directly:

e.g. `https://github.com/keybase/client/issues/new?body=Keybase%20GUI%20Version:%20${version}`